aiida_quantumespresso/workflows/pw/base.py

```diff
@@ -139,6 +169,8 @@ def validate_inputs(self):
             if num_machines is None or max_wallclock_seconds is None:
                 return self.exit_codes.ERROR_INVALID_INPUT_RESOURCES_UNDERSPECIFIED
 
+            self.set_max_seconds(max_wallclock_seconds)
```

Comment:
Why do this only if automatic_parallelization is enabled?

Reply:
No this is done only when `automatic_parallelization` is *not* enabled. The reason is because in the case of the init run, we use the default options which sets half an hour, which should be more than enough for the init run.